sections/clause_0_front_material.adoc

```diff
@@ -1,10 +1,27 @@
 [abstract]
 == Abstract
 
-This best practice document on UML to JSON encoding rules is established as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+This best practice document on UML to JSON encoding rules is initiated as an initiative of Geonovum. The aim is to come to a standardized encoding from UML to JSON, in order to achieve technical interoperability in the chain from conceptual models to JSON implementation. In this document, JSON implementation includes plain JSON, GeoJSON and JSON-FG.
+
+Application schema in UML are used to model geospatial information as part of data specifications defining information and data content of domain related universe of discourse. In the geospatial domain the ISO 19103:2015 and ISO 19109:2015 UML profile is used, which therefore is also used in this document. Application schema operate at the conceptual level. At the implementation or data level JSON is one of the major data encodings used by current web applications. The UML to JSON encoding rules defined in the context of this document is a best practice. The aim of this document is to support the development of an international standard in this domain.
```

Comment:
Suggestion: "Application schemas in UML are used to model geospatial information for a given domain, as part of data specifications defining information and data content of a relevant universe of discourse. In the geospatial domain, UML profiles are defined by <<ref_iso19103>> and <<ref_iso19109>>. These profiles are also used in this document. Application schemas operate at the conceptual level. At the implementation or data level, JSON is one of the major data encodings used by current web applications. The UML to JSON encoding rules defined in the context of this document represent a best practice."

Not entirely sure about the last sentence: "The aim of this document is to support the development of an international standard in this domain." Right now, the specification is intended to become an OGC Best Practice document. In the future, the encoding rules may become an international standard (OGC/ISO). Should we omit the sentence, or rephrase it? For example: "Eventually, this document may also support the development of an international standard for the conversion of UML to JSON (Schema) in the geospatial domain."

Reply:
rephrasing is preferred

Reply:
done